### ISSUE TITLE
LOW - cambio en Detalles

### ISSUE THREAD

**Issue by etoya85** [jimbomilk/addmeetooMng#15]
Un juego está **_en marcha_**, entro en los **_detalles_**, hago algun cambio y lo guardo. Los cambios no salen a la pantalla si no le doy a _**Preview**_. Esto o hay que escribir en algun sitio del sistema o cambiar para que salgan los cambios despues de guardarlos. Cómo piensas?

**Comment by jimbomilk** [comment]
No , el funcionamiento es que las pantallas se refrescan con nuevos anuncios y juegos cada 10 minutos. Mientras tanto aunque cambies algo no se ve reflejado los cambios hasta que pase ese tiempo.

**Comment by etoya85** [comment]
sí, ya lo sé, por eso esperaba lo que hace falta y no, no salen. es un bug
